<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nInterferometry</class>
 <widget class="QMainWindow" name="nInterferometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>497</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Interferometry</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,0,0,0,0">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="4" column="0">
     <widget class="QGroupBox" name="Zero">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Zero phase point &amp;&amp; rotation</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="posZero">
         <property name="text">
          <string/>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="offset">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QDoubleSpinBox" name="rotAngle">
         <property name="toolTip">
          <string>rotate angle</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="posZeroButton">
         <property name="toolTip">
          <string>click on the win to grab a point</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/neutrino.qrc">
           <normaloff>:/icons/center.png</normaloff>:/icons/center.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="posZeroY">
         <property name="toolTip">
          <string>zero y position</string>
         </property>
         <property name="minimum">
          <number>-99999</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="posZeroX">
         <property name="toolTip">
          <string>zero x position</string>
         </property>
         <property name="minimum">
          <number>-99999</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Correlation and carrier</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0,0,0,0">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="angleCarrier">
         <property name="toolTip">
          <string>orientation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string> Ëš</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QDoubleSpinBox" name="weightCarrier">
         <property name="toolTip">
          <string>Frequency weight (&lt;0 for lower freq; &gt;0 for higher freq)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-9.900000000000000</double>
         </property>
         <property name="maximum">
          <double>9.900000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="widthCarrier">
         <property name="toolTip">
          <string>Interfringe</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="correlation">
         <property name="toolTip">
          <string>Correlation of fringes</string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="image2">
      <property name="title">
       <string>Shot</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="AbelGroup">
      <property name="title">
       <string>Abel</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0,0,0,0">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="4">
           <widget class="QSpinBox" name="abelY">
            <property name="toolTip">
             <string>zero y position</string>
            </property>
            <property name="minimum">
             <number>-999999</number>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QToolButton" name="abelButton">
            <property name="toolTip">
             <string>click on the win to grab a point</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/neutrino.qrc">
              <normaloff>:/icons/center.png</normaloff>:/icons/center.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="abel">
            <property name="toolTip">
             <string>Activate Abel</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="imgRes">
            <property name="toolTip">
             <string>Magnification [Âµm/px]</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="probeLambda">
            <property name="toolTip">
             <string>Probe wavelength [nm]</string>
            </property>
            <property name="text">
             <string>532</string>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QToolButton" name="symmetric">
            <property name="toolTip">
             <string>Symmetrize</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/neutrino.qrc">
              <normaloff>:/icons/flipUpDown.png</normaloff>:/icons/flipUpDown.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="image1">
      <property name="title">
       <string>Reference</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="methodBox">
      <property name="title">
       <string>Unwrap &amp;&amp; quality cutoff</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,0,0,0,0">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="method">
         <property name="toolTip">
          <string>Unwrap strategy</string>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Simple H+V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Simple V+H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Goldstein</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Miguel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Miguel+Quality</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quality</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="useBarrier">
         <property name="text">
          <string/>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="lineBarrier">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/neutrino.qrc">
           <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="useRegion">
         <property name="text">
          <string/>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="region">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/neutrino.qrc">
           <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadPref"/>
   <addaction name="actionSavePref"/>
   <addaction name="separator"/>
   <addaction name="actionRect"/>
   <addaction name="separator"/>
   <addaction name="actionCarrier"/>
   <addaction name="actionDoAll"/>
  </widget>
  <action name="actionRect">
   <property name="icon">
    <iconset>
     <normaloff>:icons//rect.png</normaloff>:icons//rect.png</iconset>
   </property>
   <property name="text">
    <string>rect</string>
   </property>
   <property name="toolTip">
    <string>Show wavelet region</string>
   </property>
  </action>
  <action name="actionSavePref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePref.png</normaloff>:icons//savePref.png</iconset>
   </property>
   <property name="text">
    <string>savePref</string>
   </property>
   <property name="toolTip">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionLoadPref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//loadPref.png</normaloff>:icons//loadPref.png</iconset>
   </property>
   <property name="text">
    <string>loadPref</string>
   </property>
   <property name="toolTip">
    <string>Load Preferences</string>
   </property>
  </action>
  <action name="actionCarrier">
   <property name="icon">
    <iconset resource="../resources/neutrino.qrc">
     <normaloff>:/icons/refresh2.png</normaloff>:/icons/refresh2.png</iconset>
   </property>
   <property name="text">
    <string>Carrier</string>
   </property>
   <property name="toolTip">
    <string>Guess carrier interfringe and angle</string>
   </property>
  </action>
  <action name="actionDoAll">
   <property name="icon">
    <iconset resource="../resources/neutrino.qrc">
     <normaloff>:/icons/autoRefresh.png</normaloff>:/icons/autoRefresh.png</iconset>
   </property>
   <property name="text">
    <string>doAll</string>
   </property>
   <property name="toolTip">
    <string>Interferometry, unwrap</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>correlation</tabstop>
  <tabstop>widthCarrier</tabstop>
  <tabstop>angleCarrier</tabstop>
  <tabstop>weightCarrier</tabstop>
  <tabstop>method</tabstop>
  <tabstop>useBarrier</tabstop>
  <tabstop>lineBarrier</tabstop>
  <tabstop>posZero</tabstop>
  <tabstop>posZeroX</tabstop>
  <tabstop>posZeroY</tabstop>
  <tabstop>posZeroButton</tabstop>
  <tabstop>rotAngle</tabstop>
  <tabstop>abel</tabstop>
  <tabstop>probeLambda</tabstop>
  <tabstop>imgRes</tabstop>
  <tabstop>abelY</tabstop>
  <tabstop>abelButton</tabstop>
  <tabstop>symmetric</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/neutrino.qrc"/>
 </resources>
 <connections/>
</ui>
